### PR TITLE
Add airline and Apache Commons CLI to Command-line Arguments Parsers

### PR DESCRIPTION
**Pull request by likel**

### CODE REVIEW

**Comment by akullpp** [comment]
Apache Commons is already included. Is there a benefit to fragment and list every subpart?

**Comment by likel** [comment]
@akullpp Already excluded. But I think many people may don't know what libs apache-commons included. Thanks.

File: README.md [hunk 1]
```diff
@@ -164,6 +164,7 @@ A curated list of awesome Java frameworks, libraries and software.
 * [args4j](http://args4j.kohsuke.org/) - Small library to parse command like arguments similar to javac.
 * [JCommander](http://jcommander.org/) - Command line arguments parsing framework with custom types and validation via implementing interfaces.
 * [JOpt Simple](http://pholser.github.io/jopt-simple/) - Simple parser that uses the POSIX getopt() and GNU getopt_long() syntaxes. Does not use annotations, uses a fluent API instead.
+* [Airline](https://github.com/airlift/airline) - Java annotation-based framework for parsing Git like command line structures.
```
Comment (akullpp) [inline README.md]: Please remove the `Java` at the beginning sind it's implied.

**Comment by likel** [comment]
A new PR #400  is created. Please review it.

**Pull request closed without merging**